dispatcher/backend/src/tests/integration/conftest.py

```diff
@@ -76,6 +79,86 @@ def database() -> mongo.Database:
     yield mongo.Database()
 
 
+class GarbageCollector:
```

Comment:
I was forced to do it because there were latent issues with tests which attempt to create a resource, succeed to do it but fails during the assertation. Cleanup was not done at many other tests were failing for no reason due to this missed cleanup.